app/views/cases/ico/_close_form.html.slim

```diff
@@ -10,6 +10,8 @@
             form_hint_text: t('.date_example'),
             today_button: {class: ''} }
 
+  = render partial: 'cases/ico/ico_final_decision', locals: {case_details: @case}
```

Comment:
The partial used here actually renders something different from what's on the designs. The design for what it renders ([from the "ICO Appeal case details closed overturned" design](https://jmwvac.axshare.com/#g=1&p=ico_appeal_case_details_closed_overturned)):

![image](https://user-images.githubusercontent.com/15608/45305941-4ca9c280-b513-11e8-8050-e576ed9a4873.png)

But the design for the [ICO Appeal case details edit ICO decision design](https://jmwvac.axshare.com/#g=1&p=ico_appeal_case_details_edit_ico_decision) looks like:

![image](https://user-images.githubusercontent.com/15608/45306071-9f837a00-b513-11e8-832f-0c851648d24e.png)

So I think we either need to sync these up, or look at a different design for this page. I think @aliuk2012 is working on this as part of CT-1848 so may be worth checking in with him.

Reply:
Cool, I'll ask Al

Reply:
We've decided to defer fixing the "ICO's decision" section for another ticket. @aliuk2012 and Mia are working on standardising the way they look.